seumich/static/seumich/styles/main.css

```diff
@@ -394,7 +394,7 @@ th[scope="row"] {
 }
 
 .footer {
-  position: absolute;
+  position: fixed;
```

Comment:
Any changes here should go into the scss. I'm going to be deleting this css file and compiling it dynamically. 

This was the only example I'd found about this but if this position fixes it that's good.

http://getbootstrap.com/docs/4.1/examples/sticky-footer-navbar/

Reply:
#32 is merged so this file would be removed and cause a conflict.

Reply:
That is ok. I intend to take similar steps with the scss. This pull can be removed.